feat(core): add getBooleanInput function (#725)
actions/toolkit: packages/core/README.md

#### Inputs/Outputs

> Action inputs can be read with `getInput` meanwhile the return type is `string`, and the inputs of the boolean type in the [YAML 1.2 "Core Schema" specification](https://yaml.org/spec/1.2/spec.html#id2804923) can be read with `getBooleanInput` meanwhile the return type is `boolean`. Outputs can be set with `setOutput` which makes them available to be mapped into inputs of other actions to ensure they are decoupled.

Reviewer: ```suggestion
Action inputs can be read with `getInput` which returns a `string` or `getBooleanInput` which parses a boolean based on the [yaml 1.2 specification](https://yaml.org/spec/1.2/spec.html#id2804923). Outputs can be set with `setOutput` which makes them available to be mapped into inputs of other actions to ensure they are decoupled.
```
